endpoints/getting-started-grpc/client/main.go

```diff
@@ -95,7 +95,12 @@ func main() {
 	}
 	if *token != "" {
 		log.Printf("Using authentication token: %s", *token)
-		ctx = metadata.NewOutgoingContext(ctx, metadata.Pairs("Authorization", fmt.Sprintf("Bearer %s", *token)))
+		_, ok := metadata.FromOutgoingContext(ctx)
+		if ok {
+			ctx = metadata.AppendToOutgoingContext(ctx, "Authorization", fmt.Sprintf("Bearer %s", *token))
```

Comment:
From reading https://github.com/grpc/grpc-go/blob/master/Documentation/grpc-metadata.md#sending-metadata, we should use `AppendToOutgoingContext` every time, even when there is no metadata yet.

> This can be used with or without existing metadata on the context. When there is no prior metadata, metadata is added; when metadata already exists on the context, kv pairs are merged in.

Reply:
Verified that 
AppendToOutgoingContext works as you mentioned.